hypha/apply/funds/blocks.py

```diff
@@ -194,6 +194,9 @@ class ApplicationCustomFormFieldsBlock(CustomFormFieldsBlock, FormFieldsBlock):
     required_blocks = ApplicationMustIncludeFieldBlock.__subclasses__()
     single_blocks = ApplicationSingleIncludeFieldBlock.__subclasses__()
 
+    class Meta:
+        block_counts = {}
+
```

Comment:
Is this still required?

Reply:
No, we don't need it. I left it to avoid error but I just tested again by removing it and everything worked fine without this. I'll remove it and will add comments about the change as well.